src/regthreesixty.js

```diff
@@ -182,49 +205,75 @@ angular.module('reg.threesixty', [])
         element.on('touchstart mousedown', mousedown);
 
         function mousedown (event) {
-          event.preventDefault();
           pointerStartPosX = getPointerEvent(event).pageX;
+          pointerStartPosY = getPointerEvent(event).pageY;
           dragging = true;
 
-          $document.on('touchmove mousemove', mousemove);
-          $document.on('touchend mouseup', mouseup);
+          element.on('touchmove mousemove', mousemove);
+          element.on('touchend mouseup', mouseup);
         }
 
         function trackPointer(event){
-          if (ready && dragging) {
+          if (ready && dragging && !scrolling) {
+
+            var pointerEvent = getPointerEvent(event);
 
-            pointerEndPosX = getPointerEvent(event).pageX;
+            pointerEndPosX = pointerEvent.pageX;
+            pointerEndPosY = pointerEvent.pageY;
 
             if(monitorStartTime < new Date().getTime() - monitorInt) {
               var frameDiff = 0,
                 direction = scope.reverse? -1 : 1 ;
 
               pointerDistance = pointerEndPosX - pointerStartPosX;
+              var xDistanceAbs = Math.abs(pointerDistance);
+              var pointerDistanceY = Math.abs(pointerEndPosY - pointerStartPosY);
 
-              if(pointerDistance > 0){
-                frameDiff = Math.ceil((totalFrames - 1) * speedMultiplier * (pointerDistance / element[0].clientWidth));
-              }else{
-                frameDiff = Math.floor((totalFrames - 1) * speedMultiplier * (pointerDistance / element[0].clientWidth));
-              }
+              if (((!initialDrag && xDistanceAbs > 1) || (initialDrag && xDistanceAbs > 5)) &&
```

Comment:
Where are those numbers coming from? (`1` and `5`) Maybe it would be more convenient to have some kind of constants here.